blog/_posts/2023-01-24-scala-center-2023-roadmap.md

```diff
@@ -0,0 +1,247 @@
+  will write tutorials showing how to perform common programming tasks (working with JSON, writing tests, etc.).
+  We will use libraries created by the community and that do not require advanced knowledge of Scala. Last, we will
+  help the authors of those libraries to perform the usual maintenance tasks.
+- **Improve the content of the course
+  [Effective Programming in Scala](https://docs.scala-lang.org/online-courses.html#effective-programming-in-scala).**
+  We will address the feedback we received from the learners of the course to smoothen the learning curve.
+
+Our stretch goals include: update our [Spark course](https://docs.scala-lang.org/online-courses.html#scala-specialization),
+teach Scala to first year Bachelor students at EPFL, and create a new Scala course for beginners (with no experience at
+all in programming).
+
+### Developer Experience
+
+In addition to maintaining some core tools of the ecosystem (Bloop, Coursier, sbt, scala-debug-adapter), we will:
+
+- **Streamline the distribution of `cs` (Coursier).** The current setup process is not well integrated with the various
+  operating systems. There are no packages for Linux distributions, no MSI installer for Windows, and no PKG or DMG
+  installer for macOS.
+- **Reduce the number of ways to import Scala projects in IDEs.** Currently, IDEs support several ways for importing
+  Scala projects (via sbt or via a build server, which itself can be provided by bloop or the underlying build tool).
+  This is a source of confusion for the users, and it increases the maintenance costs.
+- **Make Metals work with more versions of Scala.** By creating a stable interface for the presentation compiler,
+  Metals will be less tied to the release cycles of the compiler, and it will easier to use it to develop the
+  compiler itself.
+- **Improve the Scala 3 support on Metals and IntelliJ.** We heard many developers stating that the Scala 3 support
+  in IDEs was not at the same level as Scala 2, and that is one of the blockers to migrate to Scala 3. We will
+  identify and put weigh on the main issues that currently degrade the developer experience on Scala 3 projects.
+
+Our stretch goals include: support standalone .scala files out of the box in Metals and IntelliJ (including support
+for Scala CLI configuration directives), investigate opportunities to reduce build times, reduce the maintenance costs
+of implementing the `debugSession/start` endpoint in all the build servers by running the debug server in Metals,
+update scalameta to Scala 3, publishing sbt 2, implement a Scala 2 expression evaluator in scala-debug-adapter,
+be able to choose a specific build target in Metals, make sure Scala programs are easy to deploy especially in IaaS,
+and provide TASTy trees to semantic rules in Scalafix.
+
+### Community and Contributor Experience
+
+In addition to continuing co-organizing ScalaDays, participating to conferences, publishing positive content about
```

Comment:
s/ScalaDays/Scala Days/
s/participating to/participating at/  (or "participating in")

I find this section to be a bit dismissive of community work as opposed to technical work. The community stuff is passed over quickly, and then a lot of space is given to specific technical proposals such as TASty MiMa.

For example, Scala Days is a really really big deal. It's a really big deal to the community, _especially_ since it's been so long since the last one because of the pandemic, and it's real work for the Center, too. I would suggest not passing over it so quickly. It gives the impression that the Center considers it minor, unimportant, and routine. (I'm sure that wasn't your intention!)

I've been told that a separate blog post about Scala Days is coming, but I still think it's worth highlighting more strongly in this blog post, too.

Reply:
Another example of the roadmap seeming very technical is: what about governance stuff, what's planned there?